entc/gen/template/meta.tmpl

```diff
@@ -58,8 +58,14 @@ var (
 		{{ end -}}
 		{{ if $f.Default }}
 			{{- $default := $f.DefaultName -}}
-			// {{ $default }} holds the default value on creation for the {{ $f.Name }} field.
-			{{ $default }} = {{ $desc }}.Default.({{ if $f.IsTime }}func() {{ end }}{{ $f.Type }})
+			{{ if $f.IsEnum }}
+			    {{ $enum := trimPackage $f.Type.String $.Package }}
+			    // {{ $default }} holds the default value on creation for the {{ $f.Name }} field.
+			    {{ $default }} = {{ $enum }}({{ $desc }}.Default.(string))
```

Comment:
I'll prefer to use one of the constants that are defined in this file (yeah, enum is a pretty special here).

Reply:
I'm afraid I don't get what you mean here.